src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -88,7 +91,8 @@ object SnowflakeTelemetry {
                        success: Boolean,
                        useProxy: Boolean,
                        queryID: Option[String],
-                       throwable: Option[Throwable]): Unit =
+                       throwable: Option[Throwable],
+                       config: Option[ObjectNode]): Unit =
```

Comment:
To Reviewers:
When exception occurs for executors, OOB message is sent. The system config is included in the telemetry message.

Reply:
Why not send config via in-band telemetry and just once ? This seems like a lot of repetition. I guess it's just for convenience of querying ?

Reply:
Also, why pass this in for each call. The config is static, correct ?